assets/javascripts/test_result.js

```diff
@@ -586,6 +586,20 @@ function setCurrentPreviewFromStepLinkIfPossible(stepLink) {
   }
 }
 
+function githashToLink(value) {
+  var hashes_regex = /(\b[0-9a-f]{9}\b)/gms;
+  var githashes = value.match(hashes_regex);
+  var repo = 'https://github.com/os-autoinst/os-autoinst-distri-opensuse/commit/';
```

Comment:
I've tested it locally. All the links to GitHub shown in the screenshot lead to a 404 page because it tries to show commits for needles within the test repository. That shows that it is problematic to hard-code the URL here and that needles needed to be distinguished from tests. (See https://github.com/os-autoinst/openQA/pull/3993#issuecomment-885628393.)

Reply:
yup this is one thing i need to improve and i wanted to ask about. i will see what i can do on Monday and i will try to reach you out for recommendations if you dont mind

Reply:
this change is ready for review.